test/controllers/RegistrationControllerSpec.scala

```diff
@@ -264,7 +264,36 @@ class RegistrationControllerSpec extends VoaPropertyLinkingSpec with MockitoSuga
 
   }
 
-  "Submitting an invalid individual form" should "return a bad request response" in {
+  "Submitting an individual form with invalid email, mobilePhone, phone, nino" should "return a bad request response" in {
+    val data = Map(
```

Comment:
You can move this to the FakeObjects class...

Reply:
I don't think this is likely to be used anywhere else, so I'm not sure putting in FakeObjects would be of benefit here.

Reply:
thanks @DanSmith98 you're right, these maps are useful only in this specific place. They belong in the test, not in the FakeObjects